pr_best_practices.py

```diff
@@ -20,13 +31,19 @@ def check_commits_contain_jira(head):
     cmd = f"git rev-list main..{head} --format='%s: %b' --no-commit-header"
     commits = os.popen(cmd).read().strip().split('\n')
     for commit in commits:
-        if re.search(r"[A-Z][A-Z0-9]+-[0-9]+", commit) is None and commit.strip():
+        # We can directly mark commits that are empty
+        if not commit.strip():
             print(f"Commit message '{commit}' should contain a Jira.")
+            continue
+
+        check_jira_issues_public(commit)
 
 
 def check_pr_description_not_empty(description):
     if description.strip():
         print("✅ Pull request description is not empty.")
+
+        check_jira_issues_public(description.strip())
```

Comment:
We don't really expect Jira tickets in descriptions, but since this is a non-blocking, purely informational test, I'm ok with adding it.